packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -83,6 +83,10 @@ const PersonalDetails = ({
         }
 
         errors.document_number = isDocumentNumberValid(document_number, document_type);
+
+        if (document_type.id !== IDV_NOT_APPLICABLE_OPTION.id && !values.confirmation_checkbox) {
+            errors.confirmation_checkbox = 'error';
```

Comment:
Rather than setting dummy error, can we set the form status value of `is_confirmed` to false?

Reply:
I need to track checkbox status with formik to properly show checked or unchecked value while witching document types, and later I need to track field error to scroll to the error field